packages/modules/web-viewer-react/src/hooks/useWebViewerInitializer.tsx

```diff
@@ -15,7 +16,7 @@ import { WebViewerProps } from "../types";
 
 export const useWebViewerInitializer = (options?: WebViewerProps) => {
   const [webViewerInitOptions, setWebViewerInitOptions] =
-    useState<WebViewerProps>();
+    useState<ItwinViewerInitializerParams>();
```

Comment:
@kckst8 I want to highlight this change and make sure you looked at this; i question it a bit. From my possibly flawed understanding, you were trying to memoize all the "base" viewer initializer props, but since the interface allowed for any kvp it didnt really pull those out. I was surprised i didnt have to update the useDesktopInitializer hook.

Here's the error which prompted me to change this.
![image](https://user-images.githubusercontent.com/11051042/141851241-56621870-b42a-4075-a0b9-d7aa30b47d1a.png)

I do think we probably want to memoize the authConfig too.

Want to get your thoughts, we can go discuss more on teams.

Reply:
This makes sense. It should be more specific. We should make the same change in the desktop initializer